api/v1/model_types.go

```diff
@@ -63,6 +61,33 @@ type ModelSpec struct {
 
 	// Env variables to be added to the server process.
 	Env map[string]string `json:"env,omitempty"`
+
+	// AutoscalingProfile to be used for autoscaling.
+	AutoscalingProfile string `json:"autoscalingProfile"`
+
+	// Autoscaling configuration.
+	// Will be set from the AutoscalingProfile if not specified.
+	Autoscaling *ModelAutoscaling `json:"autoscaling,omitempty"`
```

Comment:
can i Just set minReplicas=1 and then all other default values would be used?

So for example.
```
autoscalingProfile: "default"
autoscaling:
  minReplicas: 1
```
would result in maxReplicas being set to 100? and target request 100?

Reply:
answering myself, I believe yes due to the defaults configured in the model type itself

Reply:
`The Model "abc" is invalid: spec.autoscaling.scaleDownDelay: Required value`

The API Server will reject the request b/c if the user sets the autoscaling field they need to supply all required values. Note: the behavior as currently implemented is that if the user sets `.spec.autoscaling` then KubeAI will not override it with values from the profile. With that logic, your case here is confusing and should eventually be mitigated with a validating webhook.